crates/matrix-sdk-sled/Cargo.toml

```diff
@@ -2,12 +2,16 @@
 name = "matrix-sdk-sled"
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.60"
 
 [features]
-default = ["state-store", "crypto-store"]
+default = ["state-store"]
 
 state-store = ["matrix-sdk-base"]
-crypto-store = ["matrix-sdk-crypto"]
+crypto-store = [
+    "matrix-sdk-crypto",
+    "matrix-sdk-base?/e2e-encryption",
```

Comment:
Hmm, why does the crypto-store depend on the base crate?

Reply:
We don't, by hard. But if we remove this, and try to build as `cargo check -p matrix-sdk-sled --features crypto-store` we'll get:
```
...
    Checking matrix-sdk-sled v0.1.0 (/home/ben/Dev/matrix/matrix-rust-sdk/crates/matrix-sdk-sled)
error[E0599]: no method named `crypto_store` found for struct `StoreConfig` in the current scope
  --> crates/matrix-sdk-sled/src/lib.rs:51:56
   |
51 |         Ok(StoreConfig::new().state_store(state_store).crypto_store(crypto_store))
   |                                                        ^^^^^^^^^^^^ method not found in `StoreConfig`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `matrix-sdk-sled` due to previous error
```

That's why we have this weak-enabled here: if both are activated, we assume that you are wanting that helper activated properly, too and activate `e2e-encryption` _if_ base is included.